### PR TITLE
indicator: add SMA min() max()

### PR DESCRIPTION
**Pull request by austin362667**
in order to do some calculation which using previous values.
e.g., price normalization

### CODE REVIEW

File: pkg/indicator/sma.go [hunk 1]
```diff
@@ -30,6 +30,33 @@ func (inc *SMA) Last() float64 {
 	return inc.Values[len(inc.Values)-1]
 }
 
+func (inc *SMA) Max() float64 {
+	if len(inc.Values) == 0 {
+		return 0.0
+	}
+	max := inc.Values[len(inc.Values)-1]
```
Comment (zenixls2) [inline pkg/indicator/sma.go:37]: Why not pick the first as the default number

File: pkg/indicator/sma.go [hunk 2]
```diff
@@ -30,6 +30,33 @@ func (inc *SMA) Last() float64 {
...
+		}
+	}
+	return max
+}
+
+func (inc *SMA) Min() float64 {
+	if len(inc.Values) == 0 {
+		return 0.0
+	}
+	min := inc.Values[len(inc.Values)-1]
```
Comment (zenixls2) [inline pkg/indicator/sma.go:50]: Same here

File: pkg/indicator/sma.go (hunk 1 above)
Comment (austin362667) [reply pkg/indicator/sma.go:37]: both are ok
which one is better?

Comment (zenixls2) [reply pkg/indicator/sma.go:37]: It will be faster if you could pick up the first

**Pull request closed without merging**